### PR TITLE
List of users now shows users

### LINKED ISSUE #255: No users data available when creating cig
**Describe the bug**
When creating a cig the user will select which student the cig belongs to by selecting a student from the dropdown. Currently, it just says that there is no data available.

**To Reproduce**
1. Go to /admin/cards
2. Upload an image
3. Try to choose a student

**Expected behavior**
A list of students

### PR DESCRIPTION
**Pull request by TE4-Mattis-Abrahamsson**
## What issue did you implement or fix?
Fixes #255 

## Summary
- Added fetch job to AdminCardsView
- Changed from :data to :items, probably something that was overlooked when switching from bootstrap

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)expected)
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: frontend/src/components/CigCanvas.vue [hunk 1]
```diff
@@ -64,7 +64,7 @@ export default class CigCanvas extends Vue {
   public size: number = 1;
   public context: any = null;
 
-  public get userNames() {
+  private get userNames(): string[] {
```
Comment (TE4-jacob-lundberg) [inline frontend/src/components/CigCanvas.vue]: I believe there is a getter on it's way into the UsersModule which does exactly this. Might be better to just use that to remove unnecessary duplication. Perhaps this is a case for later.

File: frontend/src/views/AdminCardsView.vue [hunk 2]
```diff
@@ -26,12 +27,15 @@ import CanvasContainer from '@/components/CanvasContainer.vue';
   }
 })
 export default class AdminCardsViews extends Vue {
+  private images: File[] = [];
 
-  public images: File[] = [];
-
-  public onSendImages(images: any[]) {
+  private onSendImages(images: any[]) {
```
Comment (TE4-oskar-pilborg) [inline frontend/src/views/AdminCardsView.vue]: We should remove all anys

File: frontend/src/components/CigCanvas.vue (hunk 1 above)
Comment (TE4-Mattis-Abrahamsson) [reply frontend/src/components/CigCanvas.vue]: sure, however, CigCanvas is in desperate need of a rewrite so I think it deserves its own issue :-)

File: frontend/src/views/AdminCardsView.vue (hunk 2 above)
Comment (TE4-Mattis-Abrahamsson) [reply frontend/src/views/AdminCardsView.vue]: Great catch!

**Pull request merged**